tests/integration/helpers.bash

```diff
@@ -18,7 +18,7 @@ HELLO_IMAGE="$TESTDATA/hello-world.tar"
 HELLO_BUNDLE="$BATS_TMPDIR/hello-world"
 
 # CRIU PATH
-CRIU="$(which criu)"
+CRIU="$(which criu)" || true
```

Comment:
> In my understanding, the logic is (CRIU="$(which criu)") || true

I don't think that's how it works (since environment variables aren't propagated from subshells). You should instead make it something like

```
CRIU="$(which criu || true)"
```